GVRf/Framework/framework/src/main/java/org/gearvrf/GVRArmModel.java

```diff
@@ -0,0 +1,33 @@
+
+/**
+  *  https://developers.google.com/vr/unity/reference/class/gvr-arm-model-offsets
+  *  The GvrArmModel is a standard interface to interact with a scene with the controller.
+  * It is responsible for:
+  * Determining the orientation and location of the controller.
+  * Predict the location of the shoulder, elbow, wrist, and pointer.
+  *
+  */
+
+package org.gearvrf;
+import org.joml.Vector3f;
+
+public interface GVRArmModel{
```

Comment:
We can't expose an ARM model that doesn't work for all major backends. Please remove or make backend-specific.